src/goMain.ts

```diff
@@ -274,6 +274,10 @@ export function activate(ctx: vscode.ExtensionContext): void {
 		return addImport(typeof arg === 'string' ? arg : null);
 	}));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.import.workspace', (arg: string) => {
```

Comment:
What is the scenario where an arg would be passed via the command?

Reply:
Uh that's a bit of left over code, it probably wouldn't at this point, I'll remove that.